src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_BOUNDARY = 101;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(colorSupplier.getRandomColor());
+        ball.setNumber(random.nextInt(NUMBER_BOUNDARY));
```

Comment:
create a constructor for those two fields

Reply:
Could you explain me exactly what do you mean 'those two fields'???

Reply:
Probably mentor suggested to create constructor in Ball class. In this case, you can remove set methods in class Ball
`public Ball(String color, int number) {}`

Reply:
Now it's understandable. Thanks